javascript/javascript2/week2/hyf-bay/index.html

```diff
@@ -0,0 +1,60 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet">
+</head>
+<body>
+  <main>
+      <header>
+          <div class="wrapper">
+              <h1>HyfBay</h1>
+              <h2>- Buy the okay'est products</h2>
+          </div>
+      </header>
+      <section class="table-view-control">
+          <div class="search">
+              <label>What are you looking for?</label>
+              <input type="text" id="inputName">
```

Comment:
There's a [search-input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search) available if you want to use it. It doesn't change much, but it helps with semantics.